Content.Server/Lathe/LatheSystem.cs

```diff
@@ -98,13 +99,16 @@ private void OnComponentInit(EntityUid uid, LatheComponent component, ComponentI
 
         /// <summary>
         /// When someone tries to use an item on the lathe,
-        /// insert it if it's a stack and fits inside
+        /// insert it if it's a stack and fits inside and don't have anything stuck to it
         /// </summary>
         private void OnInteractUsing(EntityUid uid, LatheComponent component, InteractUsingEvent args)
         {
             if (!TryComp<MaterialStorageComponent>(uid, out var storage) || !TryComp<MaterialComponent>(args.Used, out var material))
                 return;
 
+            if (EntityManager.HasComponent<IsStuckOnEntityComponent>(args.Used) || EntityManager.HasComponent<HasEntityStuckOnComponent>(args.Used))
+                return;
+
```

Comment:
This should use a new `InsertMaterialAttemptEvent` instead to prevent spaghetti code and allow for more things than just sticky to prevent insertion.

Reply:
if I made this other event , how should I stop the original from running?

Reply:
Or do I just replace the original?

Reply:
Done